ruby_event_store-rom/lib/ruby_event_store/rom/event_repository.rb

```diff
@@ -81,6 +81,12 @@ def read(specification)
         @events.read(specification)
       end
 
+      def count(specification)
+        raise ReservedInternalName if specification.stream.name.eql?(@stream_entries.stream_entries.class::SERIALIZED_GLOBAL_STREAM_NAME)
```

Comment:
not a part of `count`

Reply:
should `count` allow for reserved name used in stream specification? maybe this check should be in `Specification` class (via `SpecificationReader` where we know repository)

Reply:
oh, if `count` omits `read` than it can check for reserved name (reminder that name will no longer be reserved after #312 materializes)

Reply:
So we should remove that check in #312 or after.